<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-018VTDCO" name="Tests.UsuarioTests" tests="1" failures="0" timestamp="2021-07-12T09:46:49 BRT" time="7.802" errors="1">
  <testcase name="tesInserirUsuario" time="7.802" classname="Tests.UsuarioTests">
    <error type="java.lang.IllegalArgumentException" message="XPath starting from / searches from root">
      <![CDATA[java.lang.IllegalArgumentException: XPath starting from / searches from root
at com.codeborne.selenide.impl.WebElementSelector.checkThatXPathNotStartingFromSlash(WebElementSelector.java:83)
at com.codeborne.selenide.impl.WebElementSelector.findElement(WebElementSelector.java:41)
at com.codeborne.selenide.impl.WebElementSelector.findElement(WebElementSelector.java:33)
at com.codeborne.selenide.impl.ElementFinder.getWebElement(ElementFinder.java:102)
at com.codeborne.selenide.impl.WebElementSource.checkCondition(WebElementSource.java:99)
at com.codeborne.selenide.impl.WebElementSource.findAndAssertElementIsInteractable(WebElementSource.java:139)
at com.codeborne.selenide.commands.Click.execute(Click.java:24)
at com.codeborne.selenide.commands.Click.execute(Click.java:16)
at com.codeborne.selenide.commands.Commands.execute(Commands.java:155)
at com.codeborne.selenide.impl.SelenideElementProxy.dispatchAndRetry(SelenideElementProxy.java:133)
at com.codeborne.selenide.impl.SelenideElementProxy.invoke(SelenideElementProxy.java:85)
at com.sun.proxy.$Proxy5.click(Unknown Source)
at Pages.UsuariosPage.consultarUsuarioTabela(UsuariosPage.java:80)
at Tests.UsuarioTests.tesInserirUsuario(UsuarioTests.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- tesInserirUsuario -->
  <system-out/>
</testsuite> <!-- Tests.UsuarioTests -->
